<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QrExportWdgt</class>
 <widget class="QWidget" name="QrExportWdgt">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>511</width>
    <height>338</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QVBoxLayout" name="vl4wdgt"/>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="lblPosition">
       <property name="text">
        <string/>
       </property>
       <property name="textInteractionFlags">
        <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QPushButton" name="pbNext">
       <property name="text">
        <string>Next</string>
       </property>
       <property name="icon">
        <iconset resource="../res.qrc">
         <normaloff>:/katynko/svg/go-next.svg</normaloff>:/katynko/svg/go-next.svg</iconset>
       </property>
      </widget>
     </item>
     <item row="1" column="5">
      <widget class="QPushButton" name="pbEnd">
       <property name="text">
        <string>End</string>
       </property>
       <property name="icon">
        <iconset resource="../res.qrc">
         <normaloff>:/katynko/svg4/go-last.svg</normaloff>:/katynko/svg4/go-last.svg</iconset>
       </property>
      </widget>
     </item>
     <item row="1" column="2" colspan="2">
      <widget class="QPushButton" name="pbPrev">
       <property name="text">
        <string>Previous</string>
       </property>
       <property name="icon">
        <iconset resource="../res.qrc">
         <normaloff>:/katynko/svg/go-previous.svg</normaloff>:/katynko/svg/go-previous.svg</iconset>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="pbFirst">
       <property name="text">
        <string>First</string>
       </property>
       <property name="icon">
        <iconset resource="../res.qrc">
         <normaloff>:/katynko/svg4/go-first.svg</normaloff>:/katynko/svg4/go-first.svg</iconset>
       </property>
      </widget>
     </item>
     <item row="1" column="7">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="6">
      <widget class="QPushButton" name="pbCancel">
       <property name="text">
        <string>Close</string>
       </property>
       <property name="icon">
        <iconset resource="../res.qrc">
         <normaloff>:/katynko/svg/dialog-cancel.svg</normaloff>:/katynko/svg/dialog-cancel.svg</iconset>
       </property>
      </widget>
     </item>
     <item row="0" column="2" colspan="6">
      <widget class="QPlainTextEdit" name="plainTextEdit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../res.qrc"/>
 </resources>
 <connections/>
</ui>
